samples/local_attestation/README.md

```diff
@@ -118,100 +126,61 @@ Let's say, we want to attest enclave 2 to enclave 1.
 
 Attesting an enclave consists of three steps:
 
-##### 1) Get an enclave's identity (target info)
+##### 1) Get an enclave's oe_verifier_get_format_settings
 
-To conduct a local attestation, both enclaves need to know each other’s identities.
-This is done by calling oe_get_target_info on the enclave 1's own report.
+If anything specific from verifier needs to be included for evidence generation,
+call oe_verifier_get_format_settings and get the required format settings.
 
 ```c
-oe_result_t oe_get_target_info(
-    const uint8_t* report,
-    size_t report_size,
-    void* target_info_buffer,
-    size_t* target_info_size);
+oe_result_t oe_verifier_get_format_settings(
+    const oe_uuid_t *format_id,
+    uint8_t **settings,
+    size_t *settings_size);
 ```
 
-On a successful return, target_info_buffer will be deposited with platform specific identity information needed for local attestation.
 
-##### 2) Generate a targeted report with the other enclave's target info (identity)
+##### 2) Generate a targeted report
 
-Inside enclave 2, call oe_get_report with the target info as opt_params. This creates a enclave_b report that' targeted at enclave 1,
+Inside enclave 2, call oe_get_evidence with uuid  = OE_FORMAT_UUID_SGX_LOCAL_ATTESTATION. This creates a enclave_b report that' targeted at enclave 1,
 that is, for enclave 1 to validate.
 
 ```c
-oe_result_t oe_get_report(
+oe_result_t oe_get_evidence(
+    const oe_uuid_t *format_id,
     uint32_t flags,
-    const uint8_t* report_data,
-    size_t report_data_size,
-    const void* opt_params,
-    size_t opt_params_size,
-    uint8_t* report_buffer,
-    size_t* report_buffer_size);
+    const void *custom_claims_buffer,
+    size_t custom_claims_buffer_size,
+    const void *optional_parameters,
+    size_t optional_parameters_size,
+    uint8_t **evidence_buffer,
+    size_t *evidence_buffer_size,
+    uint8_t **endorsements_buffer,
+    size_t *endorsements_buffer_size);
 ```
 
 ##### 3) Verify targeted report
 
-This validation consists two parts:
-
-1. Integrity of the Enclave Report
-
-    Enclave 1 can call `oe_verify_report` to validate the report originated from an Trust Execution Environment (TEE),
-    which in this case would be a valid SGX platform.
-
-    ```c
-    oe_result_t oe_verify_report(const uint8_t* report, size_t report_size, oe_report_t* parsed_report);
-    ```
-
-    At this point, Enclave 1 knows that the report originated from an enclave running in a TEE, and that the information in the report can be trusted.
-
-1. Validation of an enclave identity
```

Comment:
This section must be kept as `Attestation::attest_local_attestation_evidence()` performs enclave identity validation by examining these claims:

- "signer_id"
- "product_id"
- "security_version"
- "Public key hash" claim in custom claims ("custom_claims_buffer")

Reply:
See the version I have in PR #3520, will that text work?